Sources/SwiftDocC/Model/Rendering/RenderSectionTranslator/DeclarationsSectionTranslator.swift

```diff
@@ -79,10 +78,16 @@ struct DeclarationsSectionTranslator: RenderSectionTranslator {
     }
     
     /// Translates the given resolved topic references into encodable `OtherDeclaration`s representing the declarations of this symbol's overloads.
-    func renderOtherDeclarationsTokens(from overloads: [ResolvedTopicReference], for trait: DocumentationDataVariantsTrait,
-                                       renderNodeTranslator: inout RenderNodeTranslator) -> [DeclarationRenderSection.OtherDeclaration] {
-        var otherDeclarations = [DeclarationRenderSection.OtherDeclaration]()
-        for overloadReference in overloads {
+    func renderOtherDeclarationsTokens(from overloads: Symbol.Overloads?,
+                                       for trait: DocumentationDataVariantsTrait,
+                                       renderNodeTranslator: inout RenderNodeTranslator) -> DeclarationRenderSection.OtherDeclarations? {
+        guard let overloads else {
```

Comment:
We're still using an older Swift version that doesn't support this language feature 

```suggestion
        guard let overloads = overloads else {
```

Reply:
We could also keep the parameter non-optional and make it the callers responsibility to unwrap the value. For example, the caller could use [Optional.map(_:)](https://developer.apple.com/documentation/swift/optional/map(_:)) to only pass the variant if there's a value and return `nil` otherwise

```swift
let otherDeclarations = symbol.overloadsVariants[trait].map { variant in 
    renderOtherDeclarationsTokens(from: variant, for: trait, renderNodeTranslator: &renderNodeTranslator)
}